target/linux/bcm53xx/patches-4.14/322-ARM-dts-BCM5301X-Add-DT-for-ARRIS-SBR-AC1900P.patch

```diff
@@ -0,0 +1,118 @@
+This router has BCM4709A0 SoC, 128 MiB NAND flash (MX30LF1G18AC-TI),
+256 MiB memory and 4 x LAN and 1 x WAN ports.
+WiFi chips are BCM4360 x 2.
+USB2.0 x 1. USB3.0 x 1.
+this router cpu port only used port 8,but b53 must set port 7, is it a bug?
+see 700-b53-add-hacky-CPU-port-fixes-for-devices-not-using-p.patch
```

Comment:
Probably both ports are connected between the Switch and the SoC.

Reply:
I guess this is difference between b53 and et